backend/contributions/tests/test_required_evidence_url_types.py

```
@@ -42,26 +42,38 @@ def setUp(self):
+        self.studio_type, _ = EvidenceURLType.objects.update_or_create(
             slug='studio-contract',
-            url_patterns=[r'^https?://studio\.genlayer\.com/'],
-            is_generic=False,
-            order=2,
+            defaults=dict(
+                name='Studio Contract',
+                url_patterns=[r'^https?://studio\.genlayer\.com/'],
+                is_generic=False,
+                order=2,
+                ownership_social_account='',
+            ),
```

Ruff (0.15.18)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)